test/unittest/compiler/unittest_interpreter.cpp

```diff
@@ -185,12 +185,226 @@ TEST(nntrainerInterpreterTflite, simple_fc) {
 
   EXPECT_EQ(out, ans);
 
-  if (remove("test.tflite")) {
+  if (remove("fc_test.tflite")) {
     std::cerr << "remove ini "
-              << "test.tflite"
+              << "fc_test.tflite"
               << "failed, reason: " << strerror(errno);
   }
 }
+
+TEST(nntrainerInterpreterTflite, simple_relu) {
+
+  nntrainer::TfliteInterpreter interpreter;
+
+  auto relu0 = LayerRepresentation(
+    "activation", {"name=relu0", "activation=relu", "input_shape=1:1:1"});
+
+  auto g = makeGraph({relu0});
+
+  nntrainer::NetworkGraph ng;
+
+  for (auto &node : g) {
+    ng.addLayer(node);
+  }
+  EXPECT_EQ(ng.compile(""), ML_ERROR_NONE);
+  EXPECT_EQ(ng.initialize(), ML_ERROR_NONE);
+
+  ng.allocateTensors(nntrainer::ExecutionMode::INFERENCE);
+  interpreter.serialize(g, "relu_test.tflite");
+  ng.deallocateTensors();
+
+  tflite::ops::builtin::BuiltinOpResolver resolver;
+  std::unique_ptr<tflite::Interpreter> tf_interpreter;
+  std::unique_ptr<tflite::FlatBufferModel> model =
+    tflite::FlatBufferModel::BuildFromFile("relu_test.tflite");
+  EXPECT_NE(model, nullptr);
+  tflite::InterpreterBuilder(*model, resolver)(&tf_interpreter);
+  EXPECT_NE(tf_interpreter, nullptr);
+
+  EXPECT_EQ(tf_interpreter->AllocateTensors(), kTfLiteOk);
+
+  nntrainer::Tensor in(nntrainer::TensorDim({1, 1, 1, 1}));
+  in.setValue(2.0f);
+  nntrainer::Tensor out(nntrainer::TensorDim({1, 1, 1, 1}));
+
+  auto in_indices = tf_interpreter->inputs();
+  for (size_t idx = 0; idx < in_indices.size(); idx++) {
+    tf_interpreter->tensor(in_indices[idx])->data.raw =
+      reinterpret_cast<char *>(in.getData());
+  }
+
+  auto out_indices = tf_interpreter->outputs();
+  for (size_t idx = 0; idx < out_indices.size(); idx++) {
+    tf_interpreter->tensor(out_indices[idx])->data.raw =
+      reinterpret_cast<char *>(out.getData());
+  }
+
+  int status = tf_interpreter->Invoke();
+  EXPECT_EQ(status, TfLiteStatus::kTfLiteOk);
+
+  nntrainer::Tensor ans(nntrainer::TensorDim({1, 1, 1, 1}));
+  ans.setValue(2.0f);
+
+  EXPECT_EQ(out, ans);
+
+  if (remove("relu_test.tflite")) {
+    std::cerr << "remove ini "
+              << "relu_test.tflite"
+              << "failed, reason: " << strerror(errno);
```

Comment:
please use `strerror_r`.

Reply:
Actually, I copied this from legacy code. Could you let me know the reason of using `strerror_r` rather than `strerror`?

Reply:
There is no problem with `strerror` in this context,  it's only for svace or coverity check in the later.
But, as you mentioned, most of legacy codes are using `stderror`. It's bettor to modify whole `strerror` when the issue occurs.
sorry for bother you.